src/main/kotlin/frc/robot/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.Units
+import org.team9432.annotation.Logged
+
+interface IntakeIO {
+    val inputs: LoggedIntakeInputs
+
+    fun setSpinPower(power: Double) {
+
+    }
+
+    fun setCenterPower(power: Double) {
+
+    }
+
+    fun setAngle(angle: Measure<Angle>) {
+
+    }
+
+    fun setAnglePower(power: Double) {
+
+    }
+
+    fun reset() {
+
+    }
+
+    fun updateInputs() {
+
+    }
```

Comment:
If the functions are empty, it's better to leave both parentheses on the some row:
```suggestion
    fun setSpinPower(power: Double) {}

    fun setCenterPower(power: Double) {}

    fun setAngle(angle: Measure<Angle>) {}

    fun setAnglePower(power: Double) {}

    fun reset() {}

    fun updateInputs() {}
```